learning/tour-of-beam/frontend/integration_test/tour_page_test.dart

```diff
@@ -49,8 +51,15 @@ void main() {
       await _checkContentTreeScrollsProperly(wt);
       await _checkHighlightsSelectedUnit(wt);
       // TODO(nausharipov): fix tests
-      // await _checkRunCodeWorks(wt);
-      // await _checkResizeUnitContent(wt);
+      await _checkRunCodeWorks(wt);
+      await _checkResizeUnitContent(wt);
+
+      if (ExamplesLoader.failedToLoadExamples.isNotEmpty) {
+        final tokens = ExamplesLoader.failedToLoadExamples.join(', ');
+        print(
+          'There are some havent loaded examples with tokens: $tokens',
+        );
+      }
```

Comment:
That's from another task, do not mix it. Especially so since I have changed this part there. Please isolate the bug fix.

Reply:
Sure, removed